tensorflow/buildspec-graviton-2.7.yml

```diff
@@ -0,0 +1,39 @@
+  account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+  region: &REGION <set-$REGION-in-environment>
+  framework: &FRAMEWORK tensorflow
+  version: &VERSION 2.7.0
+  short_version: &SHORT_VERSION 2.7
+  arch_type: graviton
+
+  repository_info:
+    inference_repository: &INFERENCE_REPOSITORY
+        image_type: &INFERENCE_IMAGE_TYPE inference
+        root: !join [ *FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
+        repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", *INFERENCE_IMAGE_TYPE, "-", graviton]
+        repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]
+
+  context:
+    inference_context: &INFERENCE_CONTEXT
+      init:
+        source: docker/build_artifacts/__init__.py
+        target: __init__.py
+      dockerd-entrypoint:
+        source: docker/build_artifacts/dockerd-entrypoint.py
+        target: dockerd-entrypoint.py
+      deep_learning_container:
+        source: ../../src/deep_learning_container.py
+        target: deep_learning_container.py
+
+  images:
+    BuildEC2TensorflowCPUInferencePy3DockerImage:
+      <<: *INFERENCE_REPOSITORY
+      build: &TENSORFLOW_CPU_INFERENCE_PY3 false
+      image_size_baseline: 5000
+      device_type: &DEVICE_TYPE cpu
+      python_version: &DOCKER_PYTHON_VERSION py3
+      tag_python_version: &TAG_PYTHON_VERSION py38
+      os_version: &OS_VERSION ubuntu20.04
+      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-ec2" ]
+      docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.graviton.ec2., *DEVICE_TYPE ]
```

Comment:
Additional label ec2 added here.

Reply:
That is the old image buildspec and should be there. this insure we can build old for patching. OS version will be different. thus this yml is "2.7" and the other has no version because it's the current buildspec